x/dclauth/keeper/msg_server_approve_add_account.go

```diff
@@ -40,6 +40,15 @@ func (k msgServer) ApproveAddAccount(
 	// get pending account
 	pendAcc, _ := k.GetPendingAccount(ctx, accAddr)
 
+	// check if pending account already has reject approval from signer
+	if pendAcc.HasRejactApprovalFrom(signerAddr) {
+		return nil, sdkerrors.Wrapf(sdkerrors.ErrUnauthorized,
+			"Pending account associated with the address=%v already has approval from=%v",
```

Comment:
Message should be "`already has reject approval from`"

Reply:
Done! ✅